doc/source/myrocks/variables.rst

```diff
@@ -2164,6 +2188,59 @@ Specifies the number if table caches.
 Default value is ``6``.
 Allowed range is from ``0`` to ``19``.
 
+.. variable:: rocksdb_table_stats_background_thread_nice_value
+
+   :version 5.7.30-33: Implemented
+   :cli: ``--rocksdb-table-stats-background-thread-nice-value``
+   :dyn: Yes
+   :scope: Global
+   :vartype: Numeric
+   :default: ``19``
+   
+The nice value for index stats.
+The minimum = -20 (THREAD_PRIO_MIN)
+The maximum = 19 (THREAD_PRIO_MAX)
+
+.. variable:: rocksdb_table_stats_max_num_rows_scanned
+
+   :version 5.7.30-33: Implemented
+   :cli: ``--rocksdb-table-stats-max-num-rows-scanned``
+   :dyn: Yes
+   :scope: Global
+   :vartype: Numeric
+   :default: ``0``
+ 
+The maximum number of rows to scan in a table scan based on
+a cardinality calculation.
+The minimum is ``0`` (every modification triggers a stats recalculation).
+The maximum is ``18,446,744,073,709,551,615``.
+
+.. variable:: rocksdb_table_stats_recalc_threshold_count
+
+   :version 5.7.30-33: Implemented
+   :cli: ``--rocksdb-table-stats-recalc-threshold-count``
+   :dyn: Yes
+   :scope: Global
+   :vartype: Numeric
+   :default: ``100``
+
+The number of modified rows to trigger a stats recalculation. 
+The minimum is ``0`` (every modification triggers a stats recalculation).
```

Comment:
`every modification triggers a stats recalculation` should be removed because a stats recalculation depends on both `rocksdb_table_stats_recalc_threshold_count` and `rocksdb_table_stats_recalc_threshold_pct`:
```
  if (counter > std::max(rocksdb_table_stats_recalc_threshold_count,
                         static_cast<uint64>(
                             n_rows * rocksdb_table_stats_recalc_threshold_pct /
                             100.0))) {
```

Reply:
fixed removed text from tables_stats_recalc_threshold_count